docs/anonymity-configuration.rst

```diff
@@ -184,44 +184,50 @@ Anonymity configuration
 =======================
 
 Tahoe-LAFS provides a configuration "safety flag" for explicitly stating
-whether or not anonymity is required for a node::
+whether or not IP-address privacy is required for a node::
 
```

Comment:
Maybe link to something like https://www.torproject.org/about/overview.html.en from here? It's not a great explanation, though, of what you're really getting. For I2P maybe this one? https://geti2p.net/en/docs/how/intro

That said, I don't think either of those provide a good 1-paragraph "TL;DR" about what you're getting...i.e. that you get location- / metadata- hidden TCP connections, but that "actually anonymity" is a higher-level construct that needs app support (and likely changes in user behavior). This comes close though: https://www.torproject.org/about/overview.html.en#stayinganonymous